runners/datafusion-comet/tpcbench.py

```diff
@@ -0,0 +1,108 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import argparse
+from datetime import datetime
+import json
+from pyspark.sql import SparkSession
+import time
+
+def main(benchmark: str, data_path: str, query_path: str, iterations: int):
```

Comment:
I got:

```
pyspark.errors.exceptions.captured.AnalysisException: [UNRESOLVED_COLUMN.WITH_SUGGESTION] A column or function parameter with name `l_shipdate` cannot be resolved. Did you mean one of the following? [`lineitem`.`column_1`, `lineitem`.`column_2`, `lineitem`.`column_3`, `lineitem`.`column_4`, `lineitem`.`column_5`].; line 17 pos 1;
'Sort ['l_returnflag ASC NULLS FIRST, 'l_linestatus ASC NULLS FIRST], true
+- 'Aggregate ['l_returnflag, 'l_linestatus], ['l_returnflag, 'l_linestatus, 'sum('l_quantity) AS sum_qty#138, 'sum('l_extendedprice) AS sum_base_price#139, 'sum(('l_extendedprice * (1 - 'l_discount))) AS sum_disc_price#140, 'sum((('l_extendedprice * (1 - 'l_discount)) * (1 + 'l_tax))) AS sum_charge#141, 'avg('l_quantity) AS avg_qty#142, 'avg('l_extendedprice) AS avg_price#143, 'avg('l_discount) AS avg_disc#144, count(1) AS count_order#145L]
   +- 'Filter ('l_shipdate <= date_add(1998-12-01, -extractansiintervaldays(INTERVAL '68' DAY)))
      +- SubqueryAlias lineitem
         +- View (`lineitem`, [column_1#18L,column_2#19L,column_3#20L,column_4#21L,column_5#22L,column_6#23,column_7#24,column_8#25,column_9#26,column_10#27,column_11#28,column_12#29,column_13#30,column_14#31,column_15#32,column_16#33,column_17#34])
            +- Relation [column_1#18L,column_2#19L,column_3#20L,column_4#21L,column_5#22L,column_6#23,column_7#24,column_8#25,column_9#26,column_10#27,column_11#28,column_12#29,column_13#30,column_14#31,column_15#32,column_16#33,column_17#34] parquet
```

Reply:
The parameters are:
```
tpcbench.py --benchmark tpch --data /tmp/data --queries ../datafusion-benchmarks/tpch/queries
```

Reply:
It looks like your parquet file has the wrong column names (`column_1`, `column_2`, etc.). This is due to earlier versions of the data generation/conversion script being incorrect and not specifying the schema. This was fixed in https://github.com/apache/datafusion-benchmarks/pull/3.

Reply:
I see. Let me try it again.

Reply:
After re-generating the data, I can run this `tpcbench.py` script now. Although for comet-enabled run, it got stuck, comet-disabled run is okay.